aten/src/ATen/native/miopen/Conv_miopen.cpp

```diff
@@ -46,9 +48,11 @@ std::tuple<at::Tensor,at::Tensor,at::Tensor> miopen_convolution_backward(
 }
 
 at::Tensor miopen_convolution_transpose(
-    const at::Tensor& input, const at::Tensor& weight, const at::Tensor& bias /* optional */,
+    const Tensor& input, const Tensor& weight, const c10::optional<Tensor>& bias_opt /* optional */,
     IntArrayRef padding, IntArrayRef output_padding, IntArrayRef stride, IntArrayRef dilation,
     int64_t groups, bool benchmark, bool deterministic) {
+  const Tensor& bias = c10::value_or_else(bias_opt, [] {return Tensor();});
```

Comment:
Will clean up such cases (i.e. the impl just raises an error) in a separate pass.